styles.css

```diff
@@ -1,24 +1,52 @@
 :root {
+    --hamburger-clr: #D84282;
     --menu-bar-width: 60px;
     --menu-bar-height: 10px;
 }
 
+body {
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #333;
```

Comment:
Body kan je weglaten volgens mij. En meer vanuit je .container gaan werken vwb deze opdracht.

Reply:
De body-properties had ik toegevoegd om het hele element in het midden van de pagina te zetten. Maar niet nodig voor de opdracht inderdaad.